commercetools-models/src/main/java/io/sphere/sdk/projects/CartClassificationDraftBuilder.java

```diff
@@ -0,0 +1,42 @@
+package io.sphere.sdk.projects;
+
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Builder;
+
+import java.util.Set;
+
+public final class CartClassificationDraftBuilder extends Base implements Builder<CartClassificationDraft> {
```

Comment:
Have you tried to generate the builder classes automatically by using `@ResourceDraftValue`?

Reply:
Here the `@ResourceDraftValue` would work but just to keep homogeneity with the description above (`CartClassificationDraft.java`)

Reply:
agreed as above 👍